charybdis/server/src/transformers/csv.transformer.ts

```diff
@@ -0,0 +1,67 @@
+import { CloudData } from "../types/cloud.types";
+import { CsvDataRow, CsvDataTypeRow, CsvRunRow } from "../types/csv.types";
+import { LocalData, LocalDataType, LocalRun } from "../types/local.types";
+
+export const csvToCloudData = (
+  csvDataRow: CsvDataRow,
+  uuid: string
+): CloudData => {
+  const time = BigInt(csvDataRow.time);
+
+  const values = parseFloatArray(csvDataRow.values);
+
+  return {
```

Comment:
i dont think this is right, runId should stay runId, but id should be uuid if im not mistaken